resources/ory/charts/hydra/values.yaml

```diff
@@ -206,19 +271,126 @@ deployment:
+  # lines, adjust them as necessary, and remove the curly braces after 'annotations:'.
+  annotations: {}
+  # kubernetes.io/ingress.class: nginx
+  # kubernetes.io/tls-acme: "true"
+
+  # -- If you want to add extra sidecar containers.
+  extraContainers: {}
+  # extraContainers: |
+  #  - name: ...
+  #    image: ...
+
+  # -- If you want to add extra init containers.
+  # extraInitContainers: |
+  #  - name: ...
+  #    image: ...
+  extraInitContainers: {}
+
+  # -- If you want to add lifecycle hooks.
+  lifecycle: {}
+  # lifecycle: |
+  #   preStop:
+  #     exec:
+  #       command: [...]
+
+  # -- Set automounting of the SA token
+  automountServiceAccountToken: true
+
+  # -- Set sharing process namespace
+  shareProcessNamespace: false
+
+  # -- Specify the serviceAccountName value.
+  # In some situations it is needed to provides specific permissions to Hydra deployments
+  # Like for example installing Hydra on a cluster with a PosSecurityPolicy and Istio.
+  # Uncoment if it is needed to provide a ServiceAccount for the Hydra deployment.
+  serviceAccount:
+    # -- Specifies whether a service account should be created
+    create: true
+    # -- Annotations to add to the service account
+    annotations:
+      helm.sh/hook-weight: "0"
+      helm.sh/hook: "pre-install, pre-upgrade"
+      helm.sh/hook-delete-policy: "before-hook-creation"
+    # -- The name of the service account to use. If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  spec:
+    # -- Set job back off limit
+    backoffLimit: 10
 # Configure node affinity
 affinity: {}
 
-
-# Configures controller setup
+# -- Configures controller setup
 maester:
   enabled: true
-  # Values for the hydra admin service arguments to hydra-maester
-  adminService: {}
-    # The service name value may need to be set if you use
-    # `fullnameOverride` for the parent chart
-    # name:
-
-    # You only need to set this port if you change the value for
-    # `service.admin.port` in the parent chart
-    # port:
+
+# -- Values for the hydra admin service arguments to hydra-maester
+hydra-maester:
+  adminService:
+    # -- The service name value may need to be set if you use `fullnameOverride` for the parent chart
+    name: ""
+    # -- You only need to set this port if you change the value for `service.admin.port` in the parent chart
+    port:
+
+# -- Sidecar watcher configuration
+watcher:
+  enabled: false
+  #image: oryd/k8s-toolbox:0.0.2
+  mountFile: ""
+  # mountFile: /etc/secrets/my-secret/foo
+
+# -- PodDistributionBudget configuration
+pdb:
+  enabled: false
+  spec:
+    minAvailable: 1
+
+#  -- Parameters for the Prometheus ServiceMonitor objects.
+# Reference: https://docs.openshift.com/container-platform/4.6/rest_api/monitoring_apis/servicemonitor-monitoring-coreos-com-v1.html
+serviceMonitor:
+  # -- switch to false to prevent creating the ServiceMonitor
+  enabled: true
```

Comment:
Do we need this service? Isn't this already done in monitoring Kyma component?

Reply:
I think service monitor is still needed here